### ISSUE TITLE
[20211107] Weekly AI ArXiv 만담

### ISSUE THREAD

**Issue by jungwoo-ha** [jungwoo-ha/WeeklyArxivTalk#29]
- News
  - EMNLP 2021: 오늘 밤부터 시작. Punta Cana!!! photo by 박성준님
![image](https://user-images.githubusercontent.com/11782739/140629653-7eb8a522-bbe1-4c51-9c20-e1f1c10f87fc.png)
  - AAAI review out: Rebuttal 모두 힘내세요.
  - ICLR review will be out. 아마도 화요일
  - CVPR 2022 논문 ArXiving 소셜 미디어 정책 바뀌었어요. 10월 19일 부터 noti때까지 홍보하면 안됩니다!
  - [중국이 미국을 압도했다. 14억 인구 데이터로.](https://www.chosun.com/economy/mint/2021/11/05/QTNCOP3BKJHZ7E5KFWNM46Q2YY/)
- ArXiv
  - [Adversarial GLUE: A Multi-Task Benchmark for Robustness Evaluation of Language Models](https://arxiv.org/abs/2111.02840)
    - Large-scale LM에서 hand-crafted adversarial example의 취약성 robustness 매우 중요 (사실 NLG도 중요)
    - 14개의 adversarial attack method를 GLUE task에 적용해서 AdvGLUE 데이터셋 구축
    - 주요 요약 내용
      - Most existing adversarial attack algorithms are prone to generating invalid or ambiguous adversarial examples, with
around 90% of them either changing the original semantic meanings or misleading human annotators as well.
      - All the language models and robust training methods we tested perform poorly on AdvGLUE, with scores lagging far behind the benign accuracy.
    - 리더보드와 데이터: https://adversarialglue.github.io/
![image](https://user-images.githubusercontent.com/11782739/140641976-e228dcf7-c05f-4ba9-a101-41db3c080a2b.png)
![image](https://user-images.githubusercontent.com/11782739/140642035-86de9413-bf59-417d-8d6a-be4d960e89dd.png)
![image](https://user-images.githubusercontent.com/11782739/140642090-3f3a423a-b903-4144-b20d-9f7fc0dac322.png)
![image](https://user-images.githubusercontent.com/11782739/140642103-33a11ec6-d913-45e8-a61d-05c52e8da663.png)

  - [WaveFake: A Data Set to Facilitate Audio Deepfake Detection](https://arxiv.org/abs/2111.02813)
    - 지금까지 딥페이크 연구는 CV 도메인 위주였다. 이제 오디오 도메인도 해야한다.
    - Neural voice synthesizer 의 부작용(보이스피싱) 을 막기 위한 연구 (NeurIPS dataset track)
    - LJSpeech 로 학습한 6개의 합성모델 --> 크로스로 합성인지 아닌지 평가
    - Classifier는 MFCC쓰는 GMM 이랑 RaWNet (Conv+GRU)
    - NN-based detector가 평균적으로 성능은 높은데 variance도 크고 전통적 signal processing 기법은 robust 하다고.     
    - [데이터](https://zenodo.org/record/5642694#.YYewv2BBwuU), [코드](https://github.com/RUB-SysSec/WaveFake)
![image](https://user-images.githubusercontent.com/11782739/140642971-03a477ce-ae31-4303-adfc-8a4317c78518.png)

  - [Benchmarking Multimodal AutoML for Tabular Data with Text Fields](https://arxiv.org/abs/2111.02705)
    - Amazon에서 나온 텍스트 + 숫자 형태 tabular 데이터의 AutoML (autoGluon) 벤치마크 (NeurIPS dataset track)
    - 결론은 Multimodal Transformer + 수치 tabular 데이터 모델 앙상블한 Stack-ensemble이 최고
    - AutoGluon 베이스라인도 상당히 경쟁력 있는 편
    - Fusion 이나 앙상블하는 구체적인 방법은 appendix에서 자세하게 확인 가능
    - 데이터셋: https://github.com/sxjscience/automl_multimodal_benchmark 
![image](https://user-images.githubusercontent.com/11782739/140643064-408e8243-6723-4be5-b539-d086bd146fbf.png)

  - [MixSiam: A Mixture-based Approach to Self-supervised Representation Learning](https://arxiv.org/abs/2111.02679)
    - SimSiam을 개선한 버전
    - Data augmentation기반의 multi-view를 좀더 효과적으로 써먹기 위한 방법
    - 다른 두개의 weak aug 외에도 aug한 둘을 mix하고 aug를 encoder 태운 feature도 mix해서 mix한 둘을 contrastive로
![image](https://user-images.githubusercontent.com/11782739/140643626-c5175629-5d95-4ac1-835c-925f36d078cb.png)
![image](https://user-images.githubusercontent.com/11782739/140643707-e790d1e1-f6ee-4db8-bb72-c54bd5be5a4b.png)
![image](https://user-images.githubusercontent.com/11782739/140643643-53434b76-8c63-408c-8640-a0f432b08fc5.png)

  - [When Does Contrastive Learning Preserve Adversarial Robustness from Pretraining to Finetuning?](https://arxiv.org/abs/2111.01124)
    - NeurIPS 2021 (from MIT, IBM)
    - 제목과 같이 CL이 adversarial 에 robust할까? 특히 CSSL로 pretraining 해도 SL-finetuning 하면 그때 robustness가 약해지지 않을까?
    - ‘cross-task robustness transferability' 라고 정의
    - 메인 실험은 CIFAR10, 100 그리고 Cross로 CIFAR->STL10, 일부 ImageNet 스케일 실험있음.
    - 요약하자면
      - the design of contrastive views matters: High-frequency components of images are beneficial to improving model robustness;
      - Augmenting CL with pseudo-supervision stimulus (e.g., resorting to feature clustering) helps preserve robustness without forgetting.
      - 코드: https://github.com/LijieFan/AdvCL (아직 안만들었음 ㅡㅡ;;)     
![image](https://user-images.githubusercontent.com/11782739/140643943-d2a0071a-26c8-42b8-b13f-cdd2ae7eeaa6.png)
![image](https://user-images.githubusercontent.com/11782739/140643976-4297ef94-5be8-4df5-8e4b-d99df9fc15a3.png)

**Comment by kimyoungdo0122** [comment]
안녕하세요..! 그동안 지식 루팡만 하다가 처음으로 인사올립니다 ㅎㅎㅎ
모두의연구소에서 Enterprise AI 교육 일을 하고 있는 김영도라고 합니다!

- News
   - [State of AI Report 2021](https://www.stateof.ai/)
   - AI 컨퍼런스에 억셉되는 페이퍼 숫자의 증가
   - AI 연구자들의 커뮤니티 활동이 점점 필요해진다?! (인용, 파생연구 등 긍정적인 영향)  

![image](https://user-images.githubusercontent.com/42539153/140637511-0c40989f-26fe-49a2-9604-1f56cd3ad08a.png)
![image](https://user-images.githubusercontent.com/42539153/140637740-b4dc3ba5-414e-4e82-a852-7c3bc4ad5604.png)

   - 컨퍼런스 및 세미나 시즌!
   - [삼성 AI 포럼 후기 TF-KR](https://www.facebook.com/groups/TensorFlowKR/permalink/1611278455879873/) 
   - [SK ICT Summit 곧 다시보기가 생길 듯합니다](https://skicttechsummit.com/index.do)
   - [AI Summit 2021 SEOUL 12월 8일 ~ 9일.. 조금 비쌈](https://aisummit.co.kr/)
   - [NVIDIA GTC 11월 8일 ~ 11일](https://www.nvidia.com/ko-kr/gtc/)
   - [NAVER DEVIEW 2021 11월 24일 ~ 26일, 접수는 10일 수요일부터](https://deview.kr/2021)
   - [if KAKAO 2021 11월 16일 ~ 18일, 접수 중](https://if.kakao.com/)
   
   - [CJ그룹, 인공지능(AI) 중심 디지털 전환(DX) 한다](http://www.aitimes.kr/news/articleView.html?idxno=22996)
   - [초거대 AI 개발에 한창인 LG, 계열사별 AI 연구 상황은?](http://www.aitimes.com/news/articleView.html?idxno=141384)

**Comment by laray5959** [comment]
- ArXiv 
    - [Cross-attention conformer for context modeling in speech enhancement for ASR](https://arxiv.org/abs/2111.00127)
        -  Google

        ![crossattentionconformer](https://github.com/laray5959/FigureList/blob/main/cross_attention_conformer.PNG?raw=true)

**Comment by veritas9872** [comment]
학회 데드라인을 확인하는데 좋은 트위터 계정을 공유해드립니다.
https://twitter.com/dlcountdown
매일 주요 AI 학회의 데드라인 알람을 받을 수 있어 항상 긴장감을 놓치지 않은 채로 생활하는데 도움이 됩니다. ㅋㅋㅋ

**Comment by hollobit** [comment]
### 공정이란 가면을 쓴 AI 알고리즘
https://www.mk.co.kr/opinion/columnists/view/2021/11/1048707/
>- 사람이 개입하지 않는다고 인공지능이 공정한 건 아냐
>- 알고리즘 초기값 인간이 설정 편견·차별 없는지 따져봐야

### 2022 ICT 산업전망 컨퍼런스
http://ictconference.kr/  (11/4 ~ 11/5) 
>- 인공지능 - 사람수준으로 무한진화한다 
![2022-1](https://user-images.githubusercontent.com/998803/140647122-cf7c1449-c20e-43d7-8e5b-74450795f3f7.png)
![2022-2](https://user-images.githubusercontent.com/998803/140647133-2689d20e-5625-476e-9e27-d18e6a4dee97.png)
![2022-3](https://user-images.githubusercontent.com/998803/140647141-7917ecf7-cb76-4eff-8dee-aed6a51091ae.png)

### 코로나19 이후 더 스마트해진 인공 지능과 딥 러닝
https://www.bbntimes.com/technology/smarter-artificial-intelligence-and-deep-learning-post-covid-19
>- 그림 2020년부터 2021년까지 전 세계 기업의 인공 지능 및 딥 러닝 사용 사례
![AI그림](https://www.bbntimes.com/images/Artificial_intelligence_and_machine_learning_use_cases_for_companies_worldwide_from_2020_to_2021.png)
